### DISCUSSION TITLE
Pinned Tabs Stick to One Panel

### DISCUSSION THREAD

**Discussion by coop9653** [mbnuqw/sidebery#1616] [Q&A]
I have some pinned tabs that I visit frequently, this is why they are pinned.  But they only show above the Panel that they were in when it was created.   Shouldn't the pinned tabs show above all the panels.   Also, they should open independent of the panel,  if I open the pinned tab from the main top toolbar, it switches to the panel that the tab was pinned from.   

I am hoping there is a way to make this work.

thanks

**Comment by pajades** [comment]
There is a way to make this work.
There is a Sidebery setting that allows for Pinned Tabs to appear on all panels:
Sidebery / Settings > Settings > Tabs > Pinned Tabs > Pinned Tabs Position
